.circleci/config.yml

```diff
@@ -594,7 +593,6 @@ jobs:
             - checkout
             - attach_workspace:
                   at: .
-            - secrethub/exec:
```

Comment:
Something is missing here.
`secrethub/exec:` is a kind of wrapper of `run` to allow to execute a command with some secret (https://circleci.com/developer/orbs/orb/secrethub/cli#usage-run_command_with_secrets)

Ok to remove it, because it seems we don"t need it anymore. But it has to be replaced by a `run` step